### ISSUE TITLE
Auto imports typeshed fallback when using refactor -> move symbol

### ISSUE THREAD

**Issue by Rooba** [microsoft/pylance-release#4871]
## Environment data

-   Language Server version: 2023.9.20
-   OS and version: Debian trixie
-   Python version (& distribution if applicable, e.g. Anaconda): 3.11.5

## Code Snippet

```python
from collections import deque, Counter


class Foo:
    bar: deque
    barbar: Counter

```

## Repro Steps

1. Right Click -> Refactor -> Move Symbol To... -> Anywhere

## Expected behavior

```py
from collections import deque, Counter
```

## Actual behavior

```py
from ....vscode.extensions.ms-python.vscode-pylance-2023.9.20.dist.typeshed-fallback.stdlib.collections import Counter, deque
```

**Comment by heejaechang** [comment]
looks like when `import format` is set to `relative`, we should make sure it only apply to `user modules` not `library` or `stdlib` modules.

**Comment by heejaechang** [comment]
This issue has been fixed in prerelease version 2023.10.21, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#20231021-11-October-2023-PreRelease)
